CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-
+ 
 # Changelog
 All notable changes to this project will be documented in this file.
```

Comment:
add test which checks the number of occurances of `default_user_context(` function call